index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home - Smart City Development Konsult Ltd</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=cocogoose">
+    <script src="https://kit.fontawesome.com/a7a68c4557.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <header>
+    <nav class="navbar-black">
+      <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+      </div>
+      <div class="menu">
+        <i class="fa-brands fa-facebook-f"></i>
+        <i class="fa-brands fa-twitter"></i>
+        <ul class="header-menu">
+          <li class="header-menu-item"><a href="index.html">English</a></li>
+          <li class="header-menu-item"><a href="index.html">My Page</a></li>
+          <li class="header-menu-item"><a href="index.html">Login</a></li>
+        </ul>
+      </div>
+    </nav>
+    <nav class="navbar-white">
+      <div class="logo">
+        <a href="index.html"><img src="images/logo1.png" alt="SCDK logo"></a>
+      </div>
+      <ul class="header-white-menu">
+        <li class="header-white-menu-item"><a href="about.html">About</a></li>
+        <li class="header-white-menu-item"><a href="#program">Program</a></li>
+        <li class="header-white-menu-item"><a href="#join">Join</a></li>
+        <li class="header-white-menu-item"><a href="about.html#sponsor">Sponsor</a></li>
+        <li class="header-white-menu-item"><a href="about.html#news">News</a></li>
+      </ul>
+      <button class="white-button" type="submit">CC Campaign</button>
+    </nav>
+  </header>
+  <main>
+    <section class="headline">
+      <h1 class="headline-hello">GIS Expert world</h1>  
+      <h2 class="headline-seminar">AFRICAN GIS EXPERTS<br>INTERNATIONAL SEMINAR <br> 2022</h2>
+      <p class="headline-para">A global network of professional GIS Practisioners 
+        aimed at sharing opportunities and creating a positive change 
+        with people from over 50 countries is holding in Nigeria.</p>
+      <div class="venue">
+        <h2 class="headline-date">2022.06.22(Wed) - 24(Fri)</h2>
+        <p class="headline-venue">@Ibadan Business School, Ibadan, Nigeria.</p>
+      </div>
+    </section>
+    <!-- Program section -->
+    <section class="program" id="program">
+      <h1 class="main-program">Main Program</h1>
```

Comment:
- [ ] In order to follow best practices in semantic HTML, kindly don't use more than one `<h1>` tag in the HTML file. This will help more in SEO and file reading.